src/components/play/layer-settings/LayerSettings.js

```diff
@@ -459,6 +459,12 @@ class LayerSettings extends Component {
         this.showDeleteClearPopupButton = React.createRef()
         this.layerPopupButton = React.createRef()
         this.volumePopupButton = React.createRef()
+
+        this.muteToggle = React.createRef()
```

Comment:
Why did you need to add this? Could this be accomplished with a prop like 'isMuted'?

Reply:
this is a ref, works like an id, when used to getElement

Reply:
I could add Ref to the end of all the refs eg. muteToggleRef, I guess that would be more readable.

Reply:
Oh, just more wondering why you need this at all

I'd think the mute toggle would affect the state of the layer, then you could use that state to inform other stuff in the layer, not really sure why you need to ref the DOM

Reply:
Talked to @muganwas and the reason is because we need to only affect state if the event target is something specific. Good reason to use this!